src/commands.ts

```diff
@@ -0,0 +1,4 @@
+
+export namespace Commands {
+    export const YAML_SCHEMA_MODIFY = 'yaml.schema.modify'; // Used to modify (either add/delete) sections of JSON Schemas
```

Comment:
I am not sure about the command approach to ask for modifications from the extension. Can we just provide this as an API  extend the [schemaContributor](https://github.com/redhat-developer/vscode-yaml/blob/5b453bf389734da736d71ffb639cf73472dd9fa4/src/extension.ts#L85) object to include an API that other extensions can call?